providers/summary/scancode.js

```diff
@@ -58,16 +59,11 @@ class ScanCodeSummarizer {
 
   _summarizePackageInfo(files) {
     for (let file of files) {
-      const asserted = get(file, 'packages[0].asserted_licenses')
       // Find the first package file and treat it as the authority
-      if (asserted) {
-        const packageLicenses = addArrayToSet(
-          asserted,
-          new Set(),
-          license => license.license || license.spdx_license_key
-        )
-        return this._joinExpressions(packageLicenses)
-      }
+      const declared =
+        SPDX.normalize(get(file, 'packages[0].declared_license') || get(file, 'packages[0].declared_licensing')) ||
```

Comment:
check whether we need to map scancode license ids to SPDX ids. consider creating a `scancodeNormalize` function